connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -667,6 +667,7 @@ class SparkSession private[sql] (
    * @since 3.4.0
    */
   override def close(): Unit = {
+    client.releaseSession()
```

Comment:
I wonder if we should make it stoppable/closable ... even for dummy hosts, etc.

Reply:
Or, should we explicitly connect when we create a session? Let me match the fix to this for now in any event.